pkg/filters/filter.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package filters
+
+// Filter is an interface for filters
+type Filter interface {
```

Comment:
I don't think we need the interface here.

Reply:
It's used for the benchmark however, I agree, it should be moved to the `_tet.go` file or being unexported.

I prefer the first option.